e2e/testsuite/relayer.go

```diff
@@ -17,6 +17,7 @@ const (
 )
 
 // newCosmosRelayer returns an instance of the go relayer.
+
```

Comment:
Why would a new line be added here?

Reply:
I do not know.  Can fix it though!

Reply:
So, I think I know why and it has to do with the demonic possession of my vscode.